tox.ini

```diff
@@ -4,10 +4,11 @@ envlist = py34, py33, py32, py27, py27-configparser, pypy
 [testenv]
 passenv = HOME
 deps=
+  distribute
   pytest
   mock
 commands=
-  py.test [] tests.py
+  py.test [] tests.py tests
```

Comment:
A good idea. Do you prefer I commit a change to the branch or submit another PR?